### PR TITLE
add total mileage sum of drives and unsynced drives to drives list

### LINKED ISSUE #212: total of kilometers done for all the trips
Reported by PAH:
> When clicking on 3 horyzontal lines on the right-top corner, and click to see the registered trips, can it as well somehow show the total of kilometers done for all the trips? Is it possible even?

### PR DESCRIPTION
**Pull request by juliaanholcer**
resolves #212

### CODE REVIEW

File: frontend/src/store/index.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { modules } from './modules';
 
 export const USER = 'user';
 export const UNSYNCRONISED_DRIVES = 'unsyncedDrives';
+export const UNSYNCRONISED_DRIVES_TOTAL_MILEAGE = 'unsyncedDrivesTotalMileage';
```
Comment (adamgajzlerowicz) [inline frontend/src/store/index.js]: both line 11 and 12 are wrong.. It should be `UNSYNCHRONISED`

File: frontend/src/store/index.js [hunk 2]
```diff
@@ -39,6 +40,10 @@ const store = new Vuex.Store({
   getters: {
     [IS_ONLINE]: state => state.isOnline,
     [UNSYNCRONISED_DRIVES]: state => state.unsyncedDrives,
+    [UNSYNCRONISED_DRIVES_TOTAL_MILEAGE]: state => state.unsyncedDrives.reduce(
```
Comment (adamgajzlerowicz) [inline frontend/src/store/index.js]: UNSYNCHRONISED please

Comment (adamgajzlerowicz) [inline frontend/src/store/index.js]: Why is the reduce needed twice? It is the same function so should be extracted.
Here getter is be polluting the global space- can't it be more local to the component?

File: frontend/src/views/DrivesView.vue [hunk 3]
```diff
@@ -114,13 +126,17 @@ export default {
   methods: {
     ...mapActions(namespaces.drives, [apiActions.fetchDrives]),
     showDrive(id) {
-      this.driveVisible = id;
+      this.driveVisible = this.driveVisible === id ? null : id;
```
Comment (adamgajzlerowicz) [inline frontend/src/views/DrivesView.vue]: I think there was a separate bug for that 👍

File: frontend/src/store/index.js (hunk 2 above)
Comment (juliaanholcer) [reply frontend/src/store/index.js]: IMHO the purpose of the redux selectors is to derive the piece of state to the component and if some additional calculations are necessary they should be done by selector than in component.

File: frontend/src/store/index.js (hunk 1 above)
Comment (juliaanholcer) [reply frontend/src/store/index.js]: Sorry for that.

File: frontend/src/views/DrivesView.vue (hunk 3 above)
Comment (juliaanholcer) [reply frontend/src/views/DrivesView.vue]: I'm not sure whether there is an issue related to that. I just wanted to make it more toggle-like than only openable.

**Comment by adamgajzlerowicz** [comment]
Tested locally - looking good

**Pull request merged**